_harp/integrating_products_with_oslc/running_the_examples.md

```diff
@@ -1,55 +1,98 @@
-4.  Click **Next**.
-5.  On the Branch Selection page, select **master** and click **Next**.
-6.  For the **Destination**, select a folder for the files or accept the default of your Eclipse workspace.
-7.  Click **Finish**. `lyo.docs` will appear in the Git Repositories view.
-
-Next, import the projects:
-
-1.  In the Git Repositories view, right-click **lyo.docs** and click **Import Projects**.
-2.  In the Import Projects from Git Repository wizard, select **Import existing projects** and click **Next**.
-3.  Make sure the "Search for nested projects" option is enabled.
-4.  Select the 2 projects **lyo.docs\lyo-rest-workshop\Lab6** and **lyo.docs\lyo-rest-workshop\ninacrm** and click **Finish**.
+## Bugzilla Setup
+
+Unless you have a Bugzilla system (with admin access) you can integrate against, you need to setup a running [Bugzilla system using Docker](https://hub.docker.com/r/bugzilla/bugzilla-dev/) for the purposes of this tutorial.
+
+1. [Set up your Docker environment](https://docs.docker.com/get-started/). This is beyound the scope of this tutorial. 
+
+1. Pull the official Bugzilla image from the docker registery
+    ```bash
+	docker pull bugzilla/bugzilla-dev
+    ```
+
+1. Launch the Bugzilla container, assigning it a convenient name 'bugzilla-dev'.
+    ```bash
+	docker run -p 80:80 --name bugzilla-dev bugzilla/bugzilla-dev
+    ```
+ 
+1. This will start the docker container, with a Bugzilla server. But Bugzilla first needs to be configured before it can be used. In a new shell, login to the container
+    ```bash
+    docker exec -it bugzilla-dev bash
+    ```
+1. Now it is time to configure Bugzilla. You can configure Bugzilla by modifying the file /var/www/html/bugzilla/localconfig as desired. We will here simply assume the initial values without any modifications. If you choose to make changes to the localconfig file, make sure you adjust the commands below accordingly.
+
+1. Login to the Mysql server as root
+    ```bash
+	mysql -h localhost -u root
+    ```
+
+1. Create a new user account (with details matching those defined in the bugzilla localconfig file).
+    ```bash
+	create user 'bugs'@'localhost' identified by 'bugs';
+    ```
+
+1. Create a database 'bugs'
+    ```bash
+	create database bugs;
+    ```
+
+1. Grant the 'bugs' account all privileges to the 'bugs' database
+    ```bash
+	grant all privileges on bugs.* to bugs;
+    ```
+
+1. Exist Mysql and relogin again to make sure the 'bugs' account has access to the 'bugs' database
+    ```bash
+	exit
+    mysql -u bugs -D bugs -p
+	exit
+    ```
+	
+1. Now run the ‘checksetup.pl’ script to initlize the Bugzilla application. Among other things, you will be prompted to create an administration account for Bugzilla.
+	```bash
+    cd /var/www/html/bugzilla/
+	./checksetup.pl
```

Comment:
we should run all these steps inside that docker image and produce a docker image that does not require any config. I will see what I can do to help